src/maintainer/pinning_deps.rst

```diff
@@ -67,6 +67,36 @@ If a package is not pinned in `conda-forge-pinning <https://github.com/conda-for
 There is additional documentation on this pinning scheme in `the conda docs <https://docs.conda.io/projects/conda-build/en/latest/source/variants.html#build-variants>`_.
 
 
+.. _what_the_pins_mean:
+
+What the pins mean
+==================
+
+Pins, generally, are all about setting benchmarked versions of packages or libraries that serve as dependencies for some other packages or libraries, just so these dependent packages behave as much as the author(s) of the dependent packages would want them to behave. 
+
+So conda-forge pins are about declaring the standard version (a specific version) of a package dependency that conda-forge or a `conda recipe <https://github.com/conda-forge/staged-recipes>`_ supports, and how recipes that depend on some specific version of some libraries are set in order to not use other varying versions of the dependency.
```

Comment:
can you add this link https://conda-forge.org/blog/posts/2020-10-02-versions/ has an more outstanding overview of how conda-forge handles the pinning process?

Reply:
Awesome! I'll do that.

Many thanks for the input.